Can't load '/usr/local/software/slurm/slurm-20.11.9-rhel8/lib64/perl5/auto/Slurm/Slurm.so' for module Slurm: libperl.so.5.26: cannot open shared object file: No such file or directory at /usr/lib64/perl5/DynaLoader.pm line 190, <DATA> line 602.
 at /usr/local/software/slurm/current-rhel8/bin/generate_pbs_nodefile line 18.
Compilation failed in require at /usr/local/software/slurm/current-rhel8/bin/generate_pbs_nodefile line 18, <DATA> line 602.
BEGIN failed--compilation aborted at /usr/local/software/slurm/current-rhel8/bin/generate_pbs_nodefile line 18, <DATA> line 602.
WARNING: Logging before flag parsing goes to stderr.
W0519 08:59:59.149255 46995939251072 lazy_loader.py:50] 
The TensorFlow contrib module will not be included in TensorFlow 2.0.
For more information, please see:
  * https://github.com/tensorflow/community/blob/master/rfcs/20180907-contrib-sunset.md
  * https://github.com/tensorflow/addons
  * https://github.com/tensorflow/io (for I/O related ops)
If you depend on functionality not listed there, please file an issue.

Warning: Flow failed to import. Set the environment variable D4RL_SUPPRESS_IMPORT_ERROR=1 to suppress this message.
No module named 'flow'
Warning: CARLA failed to import. Set the environment variable D4RL_SUPPRESS_IMPORT_ERROR=1 to suppress this message.
No module named 'carla'
2022-05-19 09:00:01,942	INFO node.py:498 -- Process STDOUT and STDERR is being redirected to /tmp/ray/session_2022-05-19_09-00-01_942688_10968/logs.
2022-05-19 09:00:02,051	INFO services.py:409 -- Waiting for redis server at 127.0.0.1:11459 to respond...
2022-05-19 09:00:03,118	INFO services.py:409 -- Waiting for redis server at 127.0.0.1:52278 to respond...
2022-05-19 09:00:03,120	INFO services.py:806 -- Starting Redis shard with 10.0 GB max memory.
2022-05-19 09:00:03,138	INFO node.py:512 -- Process STDOUT and STDERR is being redirected to /tmp/ray/session_2022-05-19_09-00-01_942688_10968/logs.
2022-05-19 09:00:03,139	WARNING services.py:1294 -- Warning: Capping object memory store to 20.0GB. To increase this further, specify `object_store_memory` when calling ray.init() or ray start.
2022-05-19 09:00:03,140	INFO services.py:1442 -- Starting the Plasma object store with 20.0 GB memory using /dev/shm.
WARNING: Logging before InitGoogleLogging() is written to STDERR
E0519 09:00:03.178558 10968 io.cc:168] Connection to IPC socket failed for pathname /tmp/ray/session_2022-05-19_09-00-01_942688_10968/sockets/plasma_store, retrying 300 more times
2022-05-19 09:00:03,656	INFO tune.py:65 -- Did not find checkpoint file in ./ray_mopo/HalfCheetah/halfcheetah_mixed_capabilities_101e3.
2022-05-19 09:00:03,656	INFO tune.py:232 -- Starting a new experiment.
W0519 09:00:03.746246 46995939251072 deprecation_wrapper.py:119] From /rds/user/ajc348/hpc-work/mopo/.env/lib/python3.6/site-packages/ray/tune/logger.py:131: The name tf.VERSION is deprecated. Please use tf.version.VERSION instead.

W0519 09:00:03.746613 46995939251072 deprecation_wrapper.py:119] From /rds/user/ajc348/hpc-work/mopo/.env/lib/python3.6/site-packages/ray/tune/logger.py:136: The name tf.summary.FileWriter is deprecated. Please use tf.compat.v1.summary.FileWriter instead.

W0519 09:19:14.393364 46995939251072 deprecation_wrapper.py:119] From /rds/user/ajc348/hpc-work/mopo/.env/lib/python3.6/site-packages/ray/tune/logger.py:114: The name tf.Summary is deprecated. Please use tf.compat.v1.Summary instead.

2022-05-19 09:26:16,122	WARNING util.py:64 -- The `save_to_disk` operation took 0.8485784530639648 seconds to complete, which may be a performance bottleneck.
2022-05-19 09:26:16,126	WARNING util.py:64 -- The `process_trial` operation took 0.8617072105407715 seconds to complete, which may be a performance bottleneck.
2022-05-19 09:34:05,891	WARNING util.py:64 -- The `save_to_disk` operation took 0.5261130332946777 seconds to complete, which may be a performance bottleneck.
2022-05-19 09:34:05,894	WARNING util.py:64 -- The `process_trial` operation took 0.537839412689209 seconds to complete, which may be a performance bottleneck.
2022-05-19 09:41:52,657	WARNING util.py:64 -- The `save_to_disk` operation took 0.5337848663330078 seconds to complete, which may be a performance bottleneck.
2022-05-19 09:41:52,659	WARNING util.py:64 -- The `process_trial` operation took 0.5454425811767578 seconds to complete, which may be a performance bottleneck.
2022-05-19 09:49:37,227	WARNING util.py:64 -- The `save_to_disk` operation took 0.524298906326294 seconds to complete, which may be a performance bottleneck.
2022-05-19 09:49:37,232	WARNING util.py:64 -- The `process_trial` operation took 0.5383796691894531 seconds to complete, which may be a performance bottleneck.
2022-05-19 09:57:22,654	WARNING util.py:64 -- The `save_to_disk` operation took 1.5136759281158447 seconds to complete, which may be a performance bottleneck.
2022-05-19 09:57:22,656	WARNING util.py:64 -- The `process_trial` operation took 1.5755562782287598 seconds to complete, which may be a performance bottleneck.
slurmstepd: error: *** JOB 61382559 ON gpu-e-17 CANCELLED AT 2022-05-19T10:00:09 DUE TO TIME LIMIT ***
